extensions/wikia/WikiaRSS/WikiaRssExternalController.class.php

```diff
@@ -127,6 +133,12 @@ private function getFullItemList($items, $options) {
 			if( $item['description'] ) {
 				$text = trim(mb_convert_encoding($item['description'],$outputEncoding,$charset));
 				$text = str_replace( array("\r", "\n", "\t", '<br>'), ' ', $text );
+				$text = strip_tags( $text, '<img><a><br>' );
```

Comment:
Interesting, sounds like `removeHTMLtags()` should call `strip_tags()` internally. Not in this PR, obviously :slightly_smiling_face: